### DISCUSSION TITLE
Corsair Commander Core and Elite Capellix AIO

### DISCUSSION THREAD

**Discussion by theDoctor-exe** [Rem0o/FanControl.Releases#1380] [General]
Is there any plugin for the corsair commander core. I have an Elite Capellix 150i AIO with 6 LL 120 fans all plugged into the corsair commander core that comes with it and fanconrol doesn't detect the commander core at all. I was hoping there was a plugin for this now since I looked at the beginning of 2022 and there was a discussion on it. OpenRGB had an update that now detects the commander core and I just set it up, works great btw. Went looking for a plugin for fanconrol and I can't find one.

**Comment by theDoctor-exe** [comment]
Just to note. The Corsair Commander Core is NOT the same as the commander pro or xt

**Reply by Coolcrumble** [reply]
I am also looking for a plugin, fan control detects it however does not let me control the fans for some reason

**Reply by Rem0o** [reply]
https://github.com/EvanMulawski/FanControl.CorsairLink
